src/utilities/svgs.js

```diff
@@ -6,6 +6,7 @@ import ColonIconPath from 'assets/icons/colon.svg';
 import ContosoIconPath from 'assets/icons/contoso.svg';
 import CriticalIconPath from 'assets/icons/critical.svg';
 import DisabledIconPath from 'assets/icons/disabled.svg';
+import ElipsisIconPath from 'assets/icons/elipsis.svg';
```

Comment:
Check spelling.  It should be with 2 L's... ellipsis.

Reply:
Woops, good catch